django/utils/sys.py

```diff
@@ -0,0 +1,11 @@
+def reraise(exc_type, exc_value, tb=None):
+    """re raise excpetions captured by sys.exec_info()"""
```

Comment:
The docstring doesn't seem very descriptive. Maybe you could adapt from [six's docs](https://pythonhosted.org/six/#six.reraise)? Probably unit tests for `reraise()` should be added.

Reply:
will do Now I found upstream project

Reply:
https://github.com/benjaminp/six/blob/master/six.py#L687
There is no docstring in upstream six. Please, let me know what you want to read.